packages/terminal/src/browser/terminal-preferences.ts

```diff
@@ -97,12 +97,12 @@ export const TerminalConfigSchema: PreferenceSchema = {
             default: false
         },
         'terminal.integrated.cursorStyle': {
-            description: 'Controls the style of the terminal cursor/',
+            description: 'Controls the style of the terminal cursor.',
             enum: ['block', 'underline', 'line'],
             default: 'block'
         },
         'terminal.integrated.cursorWidth': {
-            description: 'Controls the width of the cursor when \'cursorStyle\' is set to \'line\'.',
+            markdownDescription: 'Controls the width of the cursor when `#terminal.integrated.cursorStyle` is set to `line`.',
```

Comment:
What's the `#` for?

Reply:
Thanks for the question. It is a link AFAIK. I took reference from Visual Studio Code. Do you want me to link that?

Reply:
You should not remove the `description` field and strip markdown element for it. We use `description` when hovering over the tree view. You patch improves things when in the editor, but not when using the tree-based UI.

Reply:
Thanks @marechal-p , added it back. Let me know if there is anything missing.